frontend/src/js/controllers/authCtrl.js

```diff
@@ -285,8 +285,44 @@
             }
         };
 
+        vm.oauthLoginGithub = function(){
+            vm.startLoader("Logging you in");
+            var code = location.search;
+            code =  code.split('?code=')[1]
+            console.log(code)
+
+            var parameters = {};
+            parameters.url = 'auth/oauth/github/';
+            parameters.method = 'POST';
+            parameters.data = {
+                "code": code,
+            };
+            console.log(parameters)
+            parameters.callback = {
+                onSuccess: function(response){
+                    if (response.status == 200) {
+                        utilities.storeData('userKey', response.data.key);
+                        if ($rootScope.previousState) {
+                            $state.go($rootScope.previousState);
+                            vm.stopLoader();
+                        } else {
+                            $state.go('web.dashboard');
+                        }
+                    } else {
+                        alert("Something went wrong");
+                    }
+                },
+                onError: function(response){
+                    console.log(response)
```

Comment:
Please remove console statement. Redirect the user to login page and show the error message as toast.